app/src/main/java/org/gdg/frisbee/android/chapter/InfoFragment.java

```diff
@@ -153,48 +132,9 @@ public void onPostExecute(String[] params, Person person) {
                 new ModelCache.CacheListener() {
                     @Override
                     public void onGet(Object item) {
-                        final Person chachedChapter = (Person) item;
-                        updateChapterUIFrom(chachedChapter);
-
-                        for (int chapterIndex = 0; chapterIndex < chachedChapter.getUrls().size(); chapterIndex++) {
-                            Person.Urls url = chachedChapter.getUrls().get(chapterIndex);
-                            if (url.getValue().contains("plus.google.com/")
-                                && !url.getValue().contains("communities")) {
-
-                                String org = url.getValue();
-                                try {
-                                    String id = getGPlusIdFromPersonUrl(url);
-                                    final int indexAsFinal = chapterIndex;
-                                    App.getInstance().getModelCache().getAsync(
-                                        Const.CACHE_KEY_PERSON + id,
-                                        false,
-                                        new ModelCache.CacheListener() {
-                                            @Override
-                                            public void onGet(Object item) {
-                                                addOrganizerToUI((Person) item);
-                                                if (indexAsFinal == chachedChapter.getUrls().size()) {
-                                                    setIsLoading(false);
-                                                }
-                                            }
-
-                                            @Override
-                                            public void onNotFound(String key) {
-                                                addUnknowOrganizerToUI();
-                                                if (indexAsFinal == chachedChapter.getUrls().size()) {
-                                                    setIsLoading(false);
-                                                }
-                                            }
-                                        });
-                                } catch (Exception ex) {
-                                    Snackbar snackbar = Snackbar.make(
-                                        getView(),
-                                        getString(R.string.bogus_organizer, org),
-                                        Snackbar.LENGTH_SHORT
-                                    );
-                                    ColoredSnackBar.alert(snackbar).show();
-                                }
-                            }
-                        }
+                        final Person cachedChapter = (Person) item;
+                        updateChapterUIFrom(cachedChapter);
```

Comment:
updateUIOnline tries to fetch missing organizers from google, this branch doesn't